src/main/java/com/epam/onlineshop/controllers/UserController.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.onlineshop.controllers;
+
+import com.epam.onlineshop.entities.User;
+import com.epam.onlineshop.services.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.ModelAndView;
+
+@RestController
+public class UserController {
+    @Autowired
+    private UserService userService;
+
+    @PostMapping("/registration")
+    public ModelAndView registerUser() {
+        ModelAndView modelAndView = new ModelAndView();
+        modelAndView.addObject("newUser", new User());
+        modelAndView.setViewName("regist");
+        return modelAndView;
+    }
+
+    @PostMapping("/welcome")
```

Comment:
Why POST?

Reply:
Cause we transmit user data such as password and login